### ISSUE TITLE
Training Data - scPDB

### ISSUE THREAD

**Issue by fses91** [jivankandel/PUResNet#10]
Hi,

thank you for your work.

I have a question regarding the training data (scPDB) used. Because this dataset contains only a single binding site for each protein, whereas the test datasets contains multiple binding sites. 

Did you see observe that your model only selects a single binding site most of the time? And why do you not use a training set which also contains multiple binding sites? Do you think that would improve the results?

Best regards

**Comment by jivankandel** [comment]
Sorry, for the late reply, scPDB only contains monomers only. Yes, I did observe that my model only predicts a single binding site most of the time. If you want to create a training set containing structures that contain multiple binding sites, then you have to do it manually. In the case of my training data, which is a subset of scPDB data( has pdb file processed in such a way that it has a single chain. ). A single may also have multiple binding sites, but it's rare for the single chain to have 2 active sites. If you include oligomeric structures then, I believe that it would improve your accuracy but the downfall is you have to create your dataset yourself which is a painful task.  If you are interested in both allosteric and orthosteric sites then it's a different story.  Also, I have developed a new method PUResNetV2.0, which is yet to be published this method can handle any type of monomeric or oligomeric structures. You can try at: http://nsclbio.jbnu.ac.kr/tools/jmol

**Comment by fses91** [comment]
Hi,
thanks for your answers. Yes that what was I expected, I just thought it would improve the results on the test sets, to create a dataset which contains also multiple ligands for each protein. E.g. Coach420 do have multiple bindingsites per protein.
Best regards
